apps/src/sites/code.org/pages/learn/index.js

```diff
@@ -0,0 +1 @@
+import '@cdo/apps/tutorialExplorer/tutorialExplorer';
```

Comment:
this is a smell. `tutorialExporer.js` adds things to window which then get called elsewhere. If anything adds stuff to window, it should be this entry point.